pyrdp/enum/orders.py

```diff
@@ -0,0 +1,72 @@
+"""
```

Comment:
dont forget the pyrdp header

Reply:
I'm going to do a final pass to clean up the code and make sure all of that stuff is present once the code is done.

Reply:
Fixed in e6f8bf5